frontend/packages/integration-tests-cypress/views/yaml-editor.ts

```diff
@@ -13,5 +13,13 @@ export const setEditorContent = (text: string) => {
 // initially yamlEditor loads with all grey text, finished loading when editor is color coded
 // class='mtk26' is the light blue color of property such as 'apiVersion'
 export const isLoaded = () => cy.get("[class='mtk26']").should('exist');
-export const clickSaveCreateButton = () => cy.byTestID('save-changes').click();
+// since yaml editor class mtk26 is a font class it doesn't work on an import page with no text
+// adding a check for the 1st line number, AND providing a wait allowed the load of the full component
+export const isImportLoaded = () => {
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(2000);
+  cy.get("[class='line-numbers']").should('exist');
```

Comment:
I would think the blocking `cy.get` would be sufficient here without the wait. Does it timeout?

Reply:
The monaco editor has a [const editorDidMount = React.useCallback](https://github.com/openshift/console/blob/3e2224c0f281cf3163fd8c81ea778b04b746741e/frontend/packages/console-shared/src/components/editor/YAMLEditor.tsx#L32) and I've noticed the editor initially 'loads' and then 'reloads', so cypress initially sees it's loaded, enters contents, then editor is reloaded.  I've tried to set a `window.yamlEditorLoaded=true` in this callback like cypress does [here](https://docs.cypress.io/api/commands/window#Start-tests-when-app-is-ready).  But callback happens more than once so still and issue.